californium-core/src/main/java/org/eclipse/californium/core/network/InMemoryMessageExchangeStore.java

```diff
@@ -111,14 +115,16 @@ private void startStatusLogging() {
 		final int healthStatusInterval = config.getInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 60); // seconds
 		// this is a useful health metric
 		// that could later be exported to some kind of monitoring interface
-		if (LOGGER.isTraceEnabled()) {
+		if (healthStatusInterval > 0 && LOGGER.isInfoEnabled()) {
```

Comment:
is `NetworkConfig.Keys.HEALTH_STATUS_INTERVAL` used in other places as well to control debugging output? If so, is this check for > 0 compatible with the usage in the other places?

Reply:
Currently it's used for the health_status in 
InMemoryMessageExchangeStore
InMemoryObservationStore
And in the demo-app ExtendedTestServer for the status of the "reverse observes".
So disable it everywhere with 0 seems possible without unwanted side effects.